### PR TITLE
Update the README with instructions on initialising the mongo

### PR DESCRIPTION
**Pull request by ro-tex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,56 @@ List of available parameters:
 
 - `GATSBY_API_URL`: override api url (defaults to location origin)
 
+### MongoDB Setup
+
+Mongo needs a couple of extra steps in order to start a secure cluster.
+
+* Open port 27017 to all nodes that will take part in the cluster.
```
Comment (peterjanbrone) [inline README.md:22]:
```suggestion
* Open port 27017 on all nodes that will take part in the cluster.
```

File: README.md [hunk 2]
```diff
@@ -15,6 +15,56 @@ List of available parameters:
...
+* Manually run an initialisation `docker run` with extra environment variables 
+that will initialise the admin user with a password (example below).
+* Manually add a `mgkey` file under `./docker/data/mongo` with the respective 
+secret (see Mongo's keyfile access control for details).
```
Comment (peterjanbrone) [inline README.md:26]: could link it https://docs.mongodb.com/manual/tutorial/enforce-keyfile-access-control-in-existing-replica-set/

File: README.md [hunk 3]
```diff
@@ -15,6 +15,56 @@ List of available parameters:
...
+	-v /home/user/skynet-webportal/docker/data/mongo
+```
+Cluster initialisation mongo command:
+```
+rs.initiate(
+  {
+    _id : "skynet",
+    members: [
+      { _id : 0, host : "helsinki.siasky.net:27017" },
+      { _id : 1, host : "us-va-1.siasky.net:27017" },
```
Comment (peterjanbrone) [inline README.md:62]: not sure I would explicitly mention these

**Review by peterjanbrone — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ro-tex) [reply README.md:22]: I actually meant `to` but I ended up doing `on`, haha. What I mean is that my original idea was to allow connections to 27017 only from the other nodes that are part of this replica set. But there was something wrong with that and the cluster didn't work, so I switched to "everybody can connect to 27017" for the moment. This is not great and we should revisit it later.

File: README.md (hunk 3 above)
Comment (ro-tex) [reply README.md:62]: Good point.

**Comment by ro-tex** [comment]
I'm closing this in favour of #554

**Pull request closed without merging**